examples/justfile

```
@@ -28,14 +99,16 @@ _ipfs-cmd:
         exit 1
     fi
 
-# Start start solo chain with zombienet in background
-bulletin-solo-zombienet-start:
+# Start solo chain or parachain with zombienet in background
+# Parameters:
+#   runtime - Runtime name (e.g., "bulletin-polkadot-runtime", "bulletin-westend-runtime"), taken from ../scripts/runtimes-matrix.json
+bulletin-solo-zombienet-start runtime="bulletin-polkadot-runtime":
```

I don't know, I would avoid setting here defauls, anyway we will switch to the `bulletin-westend-runtime"` probably, but let's keep this clean without default for now
```suggestion
bulletin-solo-zombienet-start runtime="":
```

---

I have a mixed feeling about this change, tbh. 

We still explicitly provide all the parameters when we call a `just` function, and defaults is one of `just` features. 
I don't have a strong opinion here. This change won't change much, sounds like a "nit", unless you have some strong arguments to back it up.

> but let's keep this clean without default for now

also there is a `default` function -- `just default`. Any preferences there?